controller/index.js

```diff
@@ -370,6 +384,20 @@ var Controller = function (agol, BaseController) {
   }
 
   /**
+   * Stub a route redirect to make testing private functions easier
+   */
+  controller.testRoute = function (req, res) {
+    controller.testMethod(req, res)
+  }
+
+  /**
+   * Stub a method for test route than can be easily wrapped
+   */
+  controller.testMethod = function (req, res) {
+    res.status(418).send('Nothing to see here.')
+  }
```

Comment:
Turns out that this is not trivial. I'd prefer to invest in the real solution of removing private methods from the controller vs removing those test methods